pkg/driver/controller.go

```diff
@@ -923,6 +923,21 @@ func (cs *controller) validateVolumeCreateReq(req *csi.CreateVolumeRequest) erro
 			"failed to handle create volume request: missing volume capabilities",
 		)
 	}
+
+	for _, volcap := range volCapabilities {
+		// VolumeCapabilities will contain volume mode
+		if mode := volcap.GetAccessMode(); mode != nil {
+			modeName := csi.VolumeCapability_AccessMode_Mode_name[int32(mode.GetMode())]
+			// At the moment we only support SINGLE_NODE_WRITER
+			if mode.GetMode() != csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER {
```

Comment:
we store the list of supported access modes in a variable over [here](https://github.com/openebs/lvm-localpv/blob/f9bebe679cb8bc6987324172827fc6c3c5807a73/pkg/driver/controller.go#L89). Can we utilise this list instead of hardcoding the supported mode here?

Reply:
Since the list is very short we can do that :+1:

Reply:
Updated with new commit